yt/tests/test_funcs.py

```diff
@@ -16,11 +16,10 @@ def test_validate_axis():
         # default geometry is cartesian
         ds = fake_amr_ds()
         ds.slice("r", 0.25)
-    desired = (
-        "Expected axis of int or char type (can be "
-        "[0, 'x', 'X', 1, 'y', 'Y', 2, 'z', 'Z']), received 'r'."
-    )
-    assert_equal(str(ex.exception)[:40], desired[:40])
+    desired = "Expected axis to be any of [0, 1, 2, 'x', 'y', 'z', 'X', 'Y', 'Z'], received 'r'"
+
+    actual = str(ex.exception)
+    assert actual == desired
```

Comment:
did you manually change this?  why?

Reply:
`ruff` automatically changed a couple strings originally formatted with the `%` operator to use the `str.format` method instead. I did a second, manual, pass to use fstrings instead and took the opportunity to slightly improve some affected error messages. Here I'm changing the test to match the -intentionally- updated message.